README.md

```diff
@@ -111,36 +112,38 @@ Here is the output of the tokenizer.
 ```
 ```python
 expression = Parser(functions={
-    'SPECIAL_UDF': SpecialUDF.from_arg_list,
+    **SpecialUdf.default_parser_mappings(),
 }).parse(tokens)[0]
 ```
 
-The expression tree produced by the parser.
+The expression tree produced by the parser:
 
 ```
-(FROM this:
- (TABLE this: x, db: ), expression:
- (SELECT expressions:
-  (COLUMN this:
-   (FUNC a:
-    (COLUMN this: a, db: , table: ), b:
-    (COLUMN this: b, db: , table: )), db: , table: )))
+(SELECT hint: , distinct: , expressions:
+  (SPECIALUDF a:
+    (COLUMN this:
+      (IDENTIFIER this: a, quoted: ), db: , table: , fields: ), b:
```

Comment:
it'd be cleaner if to_s for expression hid None args

Reply:
Ah, great idea. Ok in a follow-up PR?

Reply:
yea